From 2ce3a7553b5e840509943a4871d6debb1ee4cbba Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Fri, 26 Jan 2024 18:29:04 +0000
Subject: [PATCH 1/1] appstream metainfo: Fix appstream-builder generation

launchable desktop-id type was no longer matching after 06abbadbb501869c29076df0f0409b977869e992
causing appstream-builder to fail to generate appdata for this package.
---
 src/linux/rpi-imager.metainfo.xml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/linux/rpi-imager.metainfo.xml.in b/src/linux/rpi-imager.metainfo.xml.in
index 6b55caa..2a442f9 100644
--- a/src/linux/rpi-imager.metainfo.xml.in
+++ b/src/linux/rpi-imager.metainfo.xml.in
@@ -27,7 +27,7 @@
       download the file again.
     </p>
   </description>
-  <launchable type="desktop-id">rpi-imager.desktop</launchable>
+  <launchable type="desktop-id">org.raspberrypi.rpi-imager.desktop</launchable>
   <screenshots>
     <screenshot type="default">
       <image>http://downloads.raspberrypi.org/imager/IMAGING-UTILITY-MAIN.png</image>
-- 
2.43.0

